### PR TITLE
NoIssue-Update screengrab file

### PR DESCRIPTION
**Pull request by isabelrios**
We need that file to be updated so that we can run the screenshot tests. 
Build failed due to that: https://app.bitrise.io/build/0e31de4d-684e-403c-8dbf-1665d3bea50b#?tab=log

### CODE REVIEW

File: bitrise.yml [hunk 1]
```diff
@@ -1285,7 +1285,7 @@ trigger_map:
 - push_branch: v34.1
   workflow: xcode12-release-and-beta-nocache
 - pull_request_target_branch: main
-  workflow: RunUnitTests
+  workflow: L10nBuild
```
Comment (isabelrios) [inline bitrise.yml:1288]: ❗ Undo this change before landing

**Comment by isabelrios** [comment]
[L10nBuild fails on Bitrise](https://app.bitrise.io/build/80509740-520e-488d-93da-ce6dd2c81202#?tab=log) when launching the fastlane command. It requires the Screengrab file to be updated but I can't find the way to update it. Locally I have the same file and copying it still complains. The only solution I have found in the meantime is to manually update to latest fastlane version which is working fine in the script step (it only takes 1minute)
@AaronMT please let me know if you can think of a better solution... thanks!

**Review by AaronMT — APPROVED** [review]
